type-definitions/most.d.ts

```diff
@@ -90,11 +93,14 @@ declare interface DisposeFn {
   (): void|Promise<any>;
 }
 
-export function create<A>(f: (add: (a:A) => any, end: (x:any) => any, error: (e:Error) => any) => void|DisposeFn): Stream<A>;
+export function create<A>(f: (add: (a: A) => any, end: (x: any) => any, error: (e: Error) => any) => void|DisposeFn): Stream<A>;
 export function just<A>(a: A): Stream<A>;
 export function of<A>(a: A): Stream<A>;
+export function empty(): Stream<any>;
+export function never(): Stream<any>;
 export function from<A>(as: Iterable<A>): Stream<A>;
 export function periodic<A>(period: number, a?: A): Stream<A>;
+export function fromEvent(event: string, target: any, useCapture?: boolean): Stream<Event>;
```

Comment:
> maybe it would be best to define some interfaces

Sounds good :+1:  If we do that, and we differentiate the two signatures (by not allowing `useCapture` to be used with EventEmitter, since that's pointless), I think it'll be in a pretty good spot.